api/src/test/java/org/openmrs/api/db/ProgramWorkflowDAOTest.java

```diff
@@ -92,4 +93,17 @@ public void getProgramsByName_whenThereAreProgramsWithTheGivenName_shouldReturnA
 		Assert.assertEquals(program2, programs.get(1));
```

Comment:
Thanks for this @trustbirungi , Now i was suggesting to try using AssertEquals  or assertTrue since we are upgrading to junit5, like [here](https://github.com/openmrs/openmrs-core/blob/master/api/src/test/java/org/openmrs/customdatatype/datatype/BaseMetadataDatatypeTest.java#L32) however cc @dkayiwa , @ibacher ,@teleivo  what is your suggestion her

Reply:
This assertEquals seem alright to me.
Or do you mean another line in this PR? Since that one was not changed in this PR.

Reply:
No thanks i meant the very one. thanks, i first saw it like needed to be changed

Reply:
@sherrif10 @teleivo I'd like to suggest that this be done in a separate PR so as to maintain the atomicity of this PR.

Reply:
no change is requested so nothing to do :)